test_lib/scylla_bench_tools.py

```diff
@@ -1,27 +1,29 @@
 import logging
+import random
 
 LOGGER = logging.getLogger(__name__)
 
 
-def create_scylla_bench_table_query(compaction_strategy=None):
+def create_scylla_bench_table_query(compaction_strategy=None, seed: int = None):
     """
 
     :return: cql create table query for scylla-bench
     """
-
-    compaction_strategy_option = "AND compaction = {{'class': '{}'}};".format(
+    seed = seed or random.randint(0, 1000)
```

Comment:
it will be good to print the seed number used in the test, to be able to identify it easily, and be able to reproduce, when needed.

Reply:
fixed

Reply:
can you point me where are you printing it?